public/js/index.js

```diff
@@ -1679,12 +1679,22 @@ $(function () {
 function showDeletePopup() {
     var $this = $(this);
     var filter = "";
+
+    // Replace content to repositoy name
+    var content = $('.content').text();
+    content     = content.replace("%s",$this.data('repo-name'));
+    $('.content').html(content);
```

Comment:
just to be safe, i'd prefer if either .text() was called instead of html to set the content, or if repo-name was run through an html escaper

Reply:
if use `text()`, `strong` tag not work
![scrot_2018-01-09--1515502298_667x59](https://user-images.githubusercontent.com/16787866/34721775-ad0a488c-f576-11e7-8c20-8de470693df6.png)

Reply:
Because that is problem how it is rendered in template , it must have filter safe html set

Reply:
No, that's because of text which escapes the strong tag